src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue

```diff
@@ -298,13 +294,13 @@ const handleResetVariables = () => {
     // reset variables
     originOrder.forEach((d) => {
         if (!variableState.variableProperties[d]) return;
+        if (!originData[d]) return;
         if (isEqual(variableState.variableProperties[d], originSchemaProperties[d])) {
             initialDataWithChangedVariable[d] = originData[d];
         }
     });
-
-    variableState.variableProperties = initialPropertiesWithChangedVariable;
-    variableState.variableData = initialDataWithChangedVariable;
+    dashboardDetailState.variables_schema = { ...dashboardDetailState.variables_schema, properties: initialPropertiesWithChangedVariable };
+    dashboardDetailState.variables = initialDataWithChangedVariable;
```

Comment:
suggestion: I think this code is pretty hard to understand..🥲 What about split into two functions for reset flow?

reset order: 
1. reset schema (actually updates only use state)
2. reset variables data

For Example: 
```typescript
const resetSchema = () => {
  const originProperties = variableState.originVariablesSchema.properties;
     // variableState.order is current variable properties' name list
      variableState.order.forEach((property) => {
            updateVariablePropertyUse(property, originProperties[property].use)
      });
}

const updateVariablePropertyUse = (property: string, use: boolean) => {
  dashboardDetailState.variables_schema.properties.[property]?.use = use
}

const resetVariables = () => {
  ...
}

const handleResetVariables = () => {
  resetSchema()
  resetVariables()
}
```

[Scenario]

origin variables: {
  abc: 'abc'
}
origin schema.properties: {
  abc: { use: true }
}

After update:
current schema.properties: {
  abc: { use: false }
}
current variables: {
  abc: 'ddd'
}

After reset:
current schema.properties: {
  abc: { use: true }
}
current variables: {
  abc: 'abc'
}